.github/CODEOWNERS

```
@@ -1,4 +1,37 @@
 api/                @WordPress/openverse-api
 ingestion_server/   @WordPress/openverse-api
+nginx/              @WordPress/openverse-api
+postgres/           @WordPress/openverse-api
+sample_data/        @WordPress/openverse-api
+.flake8             @WordPress/openverse-api
+.isort.cfg          @WordPress/openverse-api
+docker-compose.yml  @WordPress/openverse-api
+load_sample_data.sh @WordPress/openverse-api
+
 frontend/           @WordPress/openverse-frontend
-*                   @WordPress/openverse-maintainers
+.eslintignore       @WordPress/openverse-frontend
+.eslintrc.js        @WordPress/openverse-frontend
+.npmrc              @WordPress/openverse-frontend
+.pnpmfile.cjs       @WordPress/openverse-frontend
+package.json        @WordPress/openverse-frontend
+pnpm-lock.yaml      @WordPress/openverse-frontend
+pnpm-workspace.yaml @WordPress/openverse-frontend
+prettier.config.js  @WordPress/openverse-frontend
+.prettierignore     @WordPress/openverse-frontend
+
+automations/                @WordPress/openverse-maintainers
+brand/                      @WordPress/openverse-maintainers
+dead_links/                 @WordPress/openverse-maintainers
+docs/                       @WordPress/openverse-maintainers
+documentation/              @WordPress/openverse-maintainers
+readme_assets/              @WordPress/openverse-maintainers
+rfcs/                       @WordPress/openverse-maintainers
+templates/                  @WordPress/openverse-maintainers
+.gitattributes              @WordPress/openverse-maintainers
+.gitignore                  @WordPress/openverse-maintainers
+.pre-commit-config.yaml     @WordPress/openverse-maintainers
+CODE_OF_CONDUCT.md          @WordPress/openverse-maintainers
+CONTRIBUTING.md             @WordPress/openverse-maintainers
+justfile                    @WordPress/openverse-maintainers
+LICENSE                     @WordPress/openverse-maintainers
+README.md
```

```suggestion
README.md
```

Is this meant to be assigned to `@WordPress/openverse-maintainers` instead of blank?

---

Right! Good catch, thanks.